scripts/bootstrap.sh.in

```diff
@@ -75,7 +99,7 @@ run_quiet() {
     local start
     start=$(date +%s)
     set +e
-    "$@" 2>&1 | tee -ia "${LOGFILE}" > "${TMPFILES}/out"
+    "$@" > >(tee -ia "${LOGFILE}" > "${TMPFILES}/out") 2>&1
```

Comment:
Commit message is wrong, it's not related to Ubuntu.
It's because you set global variables in one of the command executed here.
Should rather be something like:
```
Replace pipe by a process substitution to allow global variable assignments in the command ran.
```

Reply:
Still in commit message: `it occurs` => `it creates`

Could be worth to have a comment here on addition to the commit message actually.